cards/document-standard/component.js

```diff
@@ -48,7 +48,11 @@ class document_standardCardComponent extends BaseCard['document-standard'] {
         eventType: 'CTA_CLICK',
         eventOptions: this.addDefaultEventOptions(),
         // ariaLabel: '',
-      }
+      },
+      feedback: false, // Shows thumbs up/down buttons to provide feedback on the result card
+      feedbackTextOnSubmission: 'Thanks for the feedback!', // Text to display after a thumbs up/down is clicked
```

Comment:
I think this message should be consistent between the results cards and the DA cards. The DA cards have it as "Thank you for your feedback!". I think have consistency between these messages will also make the translation test failure go away. We already have translations for the DA cards static text.

Reply:
Makes sense. I'll change that